schema/classes/Person.yaml

```diff
@@ -0,0 +1,46 @@
+"@context":
```

Comment:
should the person have an orcid?

Reply:
i included sameAs to indicate potential other identifiers for a person. this is up for discussion as to how we support multiple identifiers for the same person. orcid is an identfier, and so is w3id, and so.